tdrs-backend/tdpservice/reports/views.py

```diff
@@ -17,15 +18,43 @@
 logger = logging.getLogger()
 
 
+class ReportFileFilter(filters.FilterSet):
+    """Filters that can be applied to GET requests as query parameters."""
+
+    # Override the generated definition for the STT field so we can require it.
+    stt = filters.NumberFilter(field_name='stt_id', required=True)
```

Comment:
Note that this is required when these filters are active. This is important to prevent users from being able to view data for STTs they do not have access to. When this is supplied in the query parameters the permission classes are able to confirm the user has the appropriate access level to see reports for the given STT. Without this being required reports from all STTs can be retrieved by omitting the parameter.